nss-day-cohort-60/rare-client-glorious-gitastrophies
# User to author rename

### To test

1. Start application and login
2. Navigate to users manager
3. Check and see if users are displaying
4. If so, click on user and check and see if proper information is displayed
